src/core/isPredOrFunc.spec.js

```diff
@@ -0,0 +1,30 @@
+const test = require('tape')
+
+const Pred = require('../Pred')
+const isNumber = require('./isNumber')
+const unit = require('./_unit')
+const isLargeNumber =
```

Comment:
insert a new line above "block"ish bits of text please. Also in test 🌽structs should be places after all requires.